commands/bootstrap/inputs.go

```diff
@@ -783,11 +840,37 @@ func getHostsInputs(resourcesPath string) ([]rest.HostConfig, error) {
 			hostConfig.Connection.Port = 22
 		}
 
-		fmt.Println("Defining key/value labels for this host, a public_address label should be defined")
+		//asking public address first
+		fmt.Println("Defining key/value labels for this host. Defining public_address value is mandatory")
 		hostConfig.Labels = make(map[string]string)
-		labelsFinished := false
-		publicAddressDefined := false
-		for !labelsFinished {
+
+		prompt = &survey.Input{
+			Message: "public_address value:"}
+
+		question = &survey.Question{
+			Name:   "value",
+			Prompt: prompt,
+		}
+		if err := survey.Ask([]*survey.Question{question}, &answer); err != nil {
+			return nil, err
+		}
+
+		hostConfig.Labels["public_address"] = answer.Value
+
+		//asking for additional key values labels
+		for {
+
+			// asking if a new label must be defined
+			promptEnd := &survey.Select{
+				Message: "Add another key/value label for this host ?:",
+				Options: []string{"yes", "no"},
+				Default: "no",
+			}
+			var reply string
+			survey.AskOne(promptEnd, &reply, nil)
```

Comment:
Is `AskOne` returned error voluntary ignored? What's the reason for doing so?

Reply:
It was like this in the code before, shall I handle the error in some way ?

Reply:
then lets mark this at solved.